activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -127,16 +127,48 @@ def test_local_cache_raw_values
     end
   end
 
+  def test_increment_initial
+    cache = lookup_store(raw: true)
+    cache.with_local_cache do
+      cache.increment("foo", 1, initial: 1)
+      assert_equal "1", cache.read("foo")
+    end
+  end
+
+  def test_increment_with_no_initial
+    cache = lookup_store(raw: true)
+    cache.with_local_cache do
+      cache.increment("foo", 2)
+      assert_equal "2", cache.read("foo")
+    end
+  end
```

Comment:
These tests could be much shorter, and IMO clearer about what they're testing:

```suggestion
  def test_increment_unset_key
    assert_equal 1, @cache.increment("foo")
    assert_equal "1", @cache.read("foo", raw: true)
  end

  def test_increment_unset_key_with_initial
    assert_equal 100, @cache.increment("foo", 1, initial: 100)
    assert_equal "100", @cache.read("foo", raw: true)
  end
```

We also need to assert on the return value, since we documented it in the examples.

Writing these tests made me realise that those examples are wrong too, the method itself returns an integer 😅